daemon/eni-multi-ip.go

```diff
@@ -278,10 +317,22 @@ func (f *eniIPFactory) popResult() (ip *types.ENIIP, err error) {
 						eni.ipAllocInhibitExpireAt = time.Now().Add(eniIPAllocInhibitTimeout)
 						eniIPLog.Infof("eni's associated vswitch %s has no available IP, set eni ipAllocInhibitExpireAt = %s",
 							eni.VSwitchID, eni.ipAllocInhibitExpireAt.Format(timeFormat))
+						ipExhaustErr := f.ipExhaustErrorLocked(result.err)
+						if ipExhaustErr != nil {
+							result.err = ipExhaustErr
+						}
 					}
 				}
 			}
 		}
+		if strings.Contains(result.err.Error(), apiErr.ErrSecurityGroupInstanceLimitExceed) {
```

Comment:
can use errors.Is() ?

Reply:
errors.Is()  need all apiErrs implement `Is` interface